presto-base-jdbc/src/main/java/io/prestosql/plugin/jdbc/BaseJdbcConfig.java

```diff
@@ -33,6 +45,31 @@
     private boolean caseInsensitiveNameMatching;
     private Duration caseInsensitiveNameMatchingCacheTtl = new Duration(1, MINUTES);
     private CredentialProviderType credentialProviderType = INLINE;
+    private String typesMappedToVarcharInclude;
+    private String typesMappedToVarcharExclude;
+    private Set<Integer> includeTypesMappingToVarchar;
+    private Set<String> includeTypeNamesMappingToVarchar;
+    private Set<Integer> excludeTypesMappingToVarchar;
+    private Set<String> excludeTypeNamesMappingToVarchar;
```

Comment:
I would extract these collection of the config classes to some new entity like `MappingToVarchar`. IMO config should not have any logic.